src/Raven.Client/Documents/Session/InMemoryDocumentSessionOperations.cs

```diff
@@ -1408,10 +1417,10 @@ internal void RegisterIncludes(BlittableJsonReaderObject includes)
             {
                 includes.GetPropertyByIndex(i, ref propertyDetails);
 
-                if (propertyDetails.Value == null)
+                if (propertyDetails.Value is BlittableJsonReaderObject json == false)
```

Comment:
Can we have sth else here?

Reply:
as far as I see no tough Oren suggest this change